src/main/java/core/basesyntax/io/impl/CsvTransactionRequestDeserializer.java

```
@@ -0,0 +1,42 @@
+package core.basesyntax.io.impl;
+
+import core.basesyntax.io.TransactionRequestDeserializer;
+import core.basesyntax.model.FruitType;
+import core.basesyntax.model.FruitTypeFactory;
+import core.basesyntax.model.TransactionRequest;
+import core.basesyntax.model.TransactionType;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.math.BigInteger;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class CsvTransactionRequestDeserializer implements TransactionRequestDeserializer {
+    private final FruitTypeFactory fruitTypeFactory;
+
+    public CsvTransactionRequestDeserializer(FruitTypeFactory fruitTypeFactory) {
+        this.fruitTypeFactory = fruitTypeFactory;
+    }
+
+    @Override
+    public List<TransactionRequest> loadFrom(Path path) {
+        try (BufferedReader reader = Files.newBufferedReader(path)) {
+            return reader.lines().skip(1).map(this::parseLine).collect(Collectors.toList());
+        } catch (IOException e) {
+            throw new RuntimeException(e);
```

loadFrom(Path path) does not validate `path` and the catch block rethrows `IOException` as `new RuntimeException(e)` with no message or path context. This violates the checklist guidance: "Don't ignore exceptions in try-catch blocks ... Let's rethrow a RuntimeException with an **informative** message and exception object." Provide an informative message containing the `path` (and optionally the line number) and pass the original exception as the cause, e.g. `throw new RuntimeException("Can't read transactions from: " + path, e);`. Also validate `path` is not null at method start.